packages/modules/pt/src/pages/citizen/MyReceipts/index.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
```

Comment:
Reciept is part for Bills, which should be common. Confirm on Monday syncup from Ajit n Sathish